charger/mennekes-compact.go

```diff
@@ -110,10 +112,16 @@ func NewMennekesCompact(ctx context.Context, uri, device, comset string, baudrat
 		conn: conn,
 	}
 
+	// check phase switching support
+	var phasesS func(int) error
+	if b, err := wb.conn.ReadHoldingRegisters(mennekesRegPhaseOptionsHW, 1); err == nil && encoding.Uint16(b) == 2 {
```

Comment:
Gleich 2 oder && 2 als Bitset?

Reply:
Wie meinen?
In der Doku steht es zumindest als Integer, nicht als Bit.